casparser/regex.py

```diff
@@ -1,8 +1,8 @@
 HEADER_RE = r"(?P<from>\d{2}-[a-zA-Z]{3}-\d{4})\s+to\s+(?P<to>\d{2}-[a-zA-Z]{3}-\d{4})"
 
 FOLIO_RE = (
-    r"Folio\s+No\s*:\s+(.+?)\s+(?:PAN\s*:\s+([A-Z]{5}\d{4}[A-Z])\s+)?"
-    r"KYC\s*:\s*(OK|NOT\s+OK)\s*(?:PAN\s*:\s*(OK|NOT\s+OK))?$"
+    r"Folio\s+No\s*:\s+([\d/\s]+)\s+.*?(?:PAN\s*:\s+([A-Z]{5}\d{4}[A-Z])\s+)?.*?"
+    r"KYC\s*:\s*(OK|NOT\s+OK)\s*.*?(?:PAN\s*:\s*(OK|NOT\s+OK))?$"
```

Comment:
This change is breaking the behavior. The problematic folios are again not being parsed now.

Reply:
Oops. I missed to commit changes from process.py. Can you please check now? 

```python
In [8]: re.search(FOLIO_RE, 'Folio No: 99999999\t\t(INR)\t\t(INR)\t\tBalance\nKYC: OK' , re.I|re.DOTALL).groups()
Out[8]: ('99999999\t', None, 'OK', None)
```

Reply:
yeah, working fine now.